components/home/list/LaunchesList.jsx

```diff
@@ -0,0 +1,36 @@
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    FlatList,
+  } from "react-native";
+  import { useRouter } from "expo-router";
+  
+  import styles from "./Landing.style";
+  
+  
+  function LaunchesList(list) {
+    const router = useRouter();
+    return (
+        <View style={styles.container}>
+        {/* TODO: Is flat list more appropriate here than mapping */}
```

Comment:
FlatList is the correct choice. It's optimized for use in mobile rendering https://blog.expo.dev/react-native-flatlist-made-easy-20fca51e0327